### PR TITLE
Repalced `unittest.TestCase.assesrtTrue` with plain assert

### LINKED ISSUE #6566: Repalce `unittest.TestCase.assesrtTrue` with plain `assert`
`unittest.TestCase.assesrtTrue` should be replaced with `assert` to get more detailed test failure information.

### Example:

```python
# test.py

import unittest


class TestRunStatus(unittest.TestCase):
    def test_assert_true_1(self):
        x = "a"
        y = "b"
        self.assertTrue(x == y)

    def test_assert_true_2(self):
        x = "a"
        y = ["b", "c"]
        self.assertTrue(x in y)

    def test_assert_1(self):
        x = "a"
        y = "b"
        assert x == y

    def test_assert_2(self):
        x = "a"
        y = ["b", "c"]
        assert x in y

```

```
pytest test.py
```

```
__________________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_1 __________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_1>

    def test_assert_1(self):
        x = "a"
        y = "b"
>       assert x == y
E       AssertionError: assert 'a' == 'b'
E         - b
E         + a

self       = <test_a.TestRunStatus testMethod=test_assert_1>
x          = 'a'
y          = 'b'

test_a.py:18: AssertionError
__________________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_2 __________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_2>

    def test_assert_2(self):
        x = "a"
        y = ["b", "c"]
>       assert x in y
E       AssertionError: assert 'a' in ['b', 'c']

self       = <test_a.TestRunStatus testMethod=test_assert_2>
x          = 'a'
y          = ['b', 'c']

test_a.py:23: AssertionError
_______________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_true_1 ________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_true_1>

    def test_assert_true_1(self):
        x = "a"
        y = "b"
>       self.assertTrue(x == y)
E       AssertionError: False is not true

self       = <test_a.TestRunStatus testMethod=test_assert_true_1>
x          = 'a'
y          = 'b'

test_a.py:8: AssertionError
_______________________________________________________________________________________________________________________________________________________ TestRunStatus.test_assert_true_2 ________________________________________________________________________________________________________________________________________________________

self = <test_a.TestRunStatus testMethod=test_assert_true_2>

    def test_assert_true_2(self):
        x = "a"
        y = ["b", "c"]
>       self.assertTrue(x in y)
E       AssertionError: False is not true

self       = <test_a.TestRunStatus testMethod=test_assert_true_2>
x          = 'a'
y          = ['b', 'c']

test_a.py:13: AssertionError
```

- `assertTrue` only tells us `True` or `False`.
- `assert` (rewritten by pytest) tells us what's different.

### PR DESCRIPTION
**Pull request by kumaryogesh17**
## Related Issues/PRs

Fixes #6566

## What changes are proposed in this pull request?

Repalced `unittest.TestCase.assesrtTrue` with plain `assert` from test_run_status.py

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Click the `Details` link on the `Preview docs` check.
2. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/pipelines`: Pipelines, Pipeline APIs, Pipeline configs, Pipeline Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/models/container/__init__.py [hunk 1]
```diff
@@ -187,9 +187,6 @@ def _serve_pyfunc(model, env_manager):
         procs.append(nginx)
 
     cpu_count = multiprocessing.cpu_count()
-    os.system("pip -V")
-    os.system("python -V")
-    os.system('python -c"from mlflow.version import VERSION as V; print(V)"')
```
Comment (harupy) [inline mlflow/models/container/__init__.py:192]: This change is unrelated. Can you revert?

File: tests/entities/test_run_status.py [hunk 2]
```diff
@@ -44,8 +44,8 @@ def test_status_mappings(self):
...
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FAILED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FINISHED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.KILLED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.SCHEDULED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.RUNNING))
+        assert RunStatus.is_terminated(RunStatus.FAILED)
+        assert RunStatus.is_terminated(RunStatus.FINISHED)
+        assert RunStatus.is_terminated(RunStatus.KILLED)
+        assert RunStatus.is_terminated(RunStatus.SCHEDULED)
+        assert RunStatus.is_terminated(RunStatus.RUNNING)
```
Comment (harupy) [inline tests/entities/test_run_status.py:51]: Are you sure this is all?

image

Comment (harupy) [inline tests/entities/test_run_status.py:51]:
```suggestion
        assert not RunStatus.is_terminated(RunStatus.SCHEDULED)
        assert not RunStatus.is_terminated(RunStatus.RUNNING)
```

**Comment by harupy** [comment]
@kumaryogesh17 Thanks for the PR, left some comments!

File: mlflow/models/container/__init__.py (hunk 1 above)
Comment (kumaryogesh17) [reply mlflow/models/container/__init__.py:192]: This was from my previous pull request , don't know why it is showing here ?

Comment (harupy) [reply mlflow/models/container/__init__.py:192]: I think it's because you created the `replace-asserttrue` branch from your previous PR's branch.

Comment (kumaryogesh17) [reply mlflow/models/container/__init__.py:192]: I have deleted old branch, I think it will solve this problem ??

Comment (harupy) [reply mlflow/models/container/__init__.py:192]: It won't because the `replace-asserttrue` branch already contains https://github.com/mlflow/mlflow/pull/6568/commits/85bc04f3c55ef2a9ce5a7766356495b6497a1805.

Comment (harupy) [reply mlflow/models/container/__init__.py:192]: You can reset commits and start from scratch or create a new PR.

Comment (kumaryogesh17) [reply mlflow/models/container/__init__.py:192]: can i delete this pull request and create a new one ?

Comment (harupy) [reply mlflow/models/container/__init__.py:192]: Yes
